src/commands/assign.js

```diff
@@ -96,29 +98,32 @@ function setPrompt() {
   }
 
   // Assigned info.
-  //
-  // Shows the number of projects that are currently assigned
-  console.log(chalk.green(`Currently assigned: ${chalk.white(assigned.length)}`));
+  if (assigned.length) {
+    const count = assigned.length === 1 ? 'one submission' : 'two submissions';
+    console.log(chalk.green(`You currently have ${count} assigned.`));
+    console.log(chalk.white.dim('Start a review in your default browser by pressing "key".'));
+  } else {
+    console.log(chalk.green('No submissions are currently assigned.'));
+  }
+  console.log(chalk.white.dim('Open the review dashboard by pressing "0".\n'));
 
   // Shows assigned projects in a table
   const submissionDetails = new Table({
     head: [
-      {hAlign: 'center', content: 'id'},
-      {hAlign: 'left', content: 'name'},
-      {hAlign: 'center', content: 'sub_id'},
-      {hAlign: 'center', content: 'time_left'}],
-    colWidths: [7, 40, 10, 15],
+      {hAlign: 'center', content: 'key'},
+      {hAlign: 'left', content: 'project name'},
+      {hAlign: 'center', content: 'time left'}],
+    colWidths: [5, 40, 15],
   });
 
   assigned
-    .forEach((submission) => {
+    .forEach((submission, idx) => {
```

Comment:
I think we just need to be sure what is the order that the assigned review will show up in the table. It can be by submission ID or by `assigned_at`. 

I don't have any idea what is the order in the Udacity's dashboard.